website/blog/2021-09-11-sql-dateadd.md

```diff
@@ -142,6 +142,3 @@ And it’s actually quite a simple 31-line macro ([source here](https://github.c
 Enjoy! FYI I've used dateadd macro in dbt-utils on BigQuery, Postgres, Redshift and Snowflake, but it likely works across most other warehouses.
 
 *Note: While `dbt_utils` doesn't support Databricks by default, you can use other packages that [implement overrides](/reference/dbt-jinja-functions/dispatch#overriding-package-macros) as a workaround.*
-
-*This [spark_utils package](https://github.com/dbt-labs/spark-utils/blob/main/macros/dbt_utils/cross_db_utils/dateadd.sql) can help you implement the override needed to add support for Databricks dateadd*
-
```

Comment:
just in case, flagging this removal too to see if it was deliberate.

Reply:
This was deliberate as the page is gone and I can't find a suitable replacement in their Git directory.  Seems it no longer exists.

Reply:
@matthewshaver is it this link: https://github.com/dbt-labs/dbt-utils/blob/eb1ec81c7bc58c3fbcd2cbcf4216b276ccd893ba/macros/cross_db_utils/dateadd.sql?

there's also another dateadd.sql file here: https://github.com/dbt-labs/spark-utils/blob/0.3.0/macros/dbt_utils/cross_db_utils/dateadd.sql but i'm not sure which is more up to date

Reply:
@mirnawong1 - You found it!  I swear I looked everywhere and all I found was that first one.  But the second should cover it.
Thank you!